src/operator/src/req_convert/insert/stmt_to_region.rs

```diff
@@ -65,6 +65,11 @@ impl<'a> StatementToRegion<'a> {
         let table_schema = table.schema();
         let table_info = table.table_info();
 
+        ensure!(
+            schema != "information_schema",
```

Comment:
Use the constant `INFORMATION_SCHEMA_NAME`

https://github.com/GreptimeTeam/greptimedb/blob/ddc7a80f56885f70e935dd9831aabb9c68d17340/src/common/catalog/src/consts.rs#L16